server/index.js

```diff
@@ -1,191 +1,60 @@
-  if (req.session.userLogin) {
-    next()
-  } else if (requestForNotLoggedIn.includes(req.url)) {
-    next()
-  }else {
-    res.sendStatus(403)
-  }
-})
-
-app.get('/todo', (req, res) =>{
-    res.send('Hello')
-})
-app.get('/tolo', async (req, res) =>{
-  let result = await getTasks(client, req.session.userLogin)
-  res.send(result.rows)
-})
-app.post('/task', async (req,res) => {
-    let result = await insertTable(client, req.body, req.session.userLogin)
-    res.send('complete')
-})
-
-app.post('/register', async (req, res) => {
-    let result = await insertRegDataUser(client, req.body)
-    let byLogin = await getLoginUserFromBase(client)
-    let checkValidPsw = checkValidRegisterPassword(req.body.password)
-  console.log(checkValidPsw)
-    if (byLogin.rows[0].login !== req.body.login){
-      res.sendStatus(200)
-  }  else {
-      res.sendStatus(403)
-  }
-})
-
-app.post('/login', async (req, res) => {
-    let result = await selectUserByLogin(client, req.body.login);
-    if (result.rows[0]){
-      let compare = comparePsw(req.body.password, result.rows[0].password)
-
-    if (compare) {
-      req.session.userLogin = result.rows[0].login
-      res.sendStatus(200)
-    } else {
-      res.sendStatus(400)
-    }
-  } else {
-    res.sendStatus(400)
-  }
-    
-})
-
-app.get('/profile', async (req, res) => {
-  if(req.session.userLogin !== null){
-    let result = await dataUser(client, req.session.userLogin)
-    res.send(result.rows[0])
-  } else {
-    res.status(400)
-  }
-})
-
-app.put('/task', async (req,res) => {
-  let result = await editTask(client, req.body)
-  res.send('complete')
-})
-
-app.get('/deleteAllOpenTask', async (req, res) => {
-  let result = await deleteAllOpenTasks(client)
-  res.send('complete')
-})
-
-app.get('/deleteAllDoneTask', async (req, res) => {
-  let result = await deleteAllDoneTasks(client)
-  res.send('complete')
-})
-
-app.delete('/deleteTask', async (req, res) => {
-  let result = await deleteTask(client, req.query.id)
-  res.send('complete')
-})
-
-app.delete('/logout', async (req, res) => {
-  await req.session.destroy()
-  res.sendStatus(200)
-})
+
+const { connectionBase } = require('./group/Settings of Server/connectBase')
+const { useCors } = require('./group/Settings of Server/useCors')
+const { cookie } = require('./group/Settings of Server/cookie')
+const { client } = require('./group/Settings of Server/client')
+const { port, listeningPort } = require('./group/Settings of Server/port')
+
+const { dist } = require('./group/Settings of Server/static')
+
+const { requestForNotLoggedIn } = require('./group/API/requestForNotLoggedIn')
+
+const { users } = require('./group/API/User/users')
+const { tasks } = require('./group/API/Tasks/tasks')
+const { logout } = require('./group/API/logut')
+
+
+app.use(morgan('dev'))
+app.use(bodyParser.json());
+useCors(app)
+
+connectionBase(client)
+cookie(app)
+listeningPort(app)
+
+dist(app, express)
```

Comment:
It's better to import express directly in file with 'dist' function. Because express is global library, you can use it everywhere by importing and it will be always the same object